src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts

```diff
@@ -185,4 +185,15 @@ export class AddRobotComponent implements OnInit, OnDestroy {
         this.messageHandlerService.showSuccess(res);
       });
   }
+
+  radioChange(num): void {
```

Comment:
the parameter 'num'? I think it is not a good way to extends function.

Reply:
The parameter 'num' is a simple mark to know which radio is checked

Reply:
I think it is not standard to use 1 or 2.If we add new permisson, do we need to add 3, 4, 5?

Reply:
modified. Thanks a lot.